x-pack/plugin/identity-provider/src/test/resources/rest-api-spec/api/idp.saml_register_sp.json

```
@@ -0,0 +1,40 @@
+{
+  "idp.saml_register_sp": {
+    "documentation": {
+      "url": "https://www.elastic.co/guide/en/elasticsearch/reference/current/idp-saml-register-sp.html"
+    },
+    "stability": "stable",
```

```suggestion
    "stability": "private",
```

Marked as `"private"`; this is an internal API that should not be exposed in clients